controllers/chat/sendSignedMessage.js

```diff
@@ -131,8 +132,44 @@ const sendSignedMesage = async (req, res) => {
       });
     }
 
-    const chat = await channel.sendMessage(baseMessage);
+    const chat = await channel.sendMessage(baseMessage, {skip_push: true});
+    let notificationPayload = {
+      title: `New Message from ${senderInfo?.user?.username}`,
+      body: `${message.substring(0, 100)}`
+    };
+    let dataPayload = {
+      channel_id: channelId,
+      user_id: req.userId,
+      messages_id: chat.message.id,
+      message,
+      message_schema: currentMessageType,
+      type: 'message.new',
+      status: 'sent',
+      is_big_message: 'false',
+      attachment: JSON.stringify(attachments) ?? '',
+      created_at: chat.message.created_at,
+      is_annoymous: 'false'
+    };
+    const dataSizeInBytes = Buffer.byteLength(JSON.stringify(dataPayload), 'utf8');
+    const dataSizeInKilobytes = dataSizeInBytes / 1024;
+    if (dataSizeInKilobytes > 4) {
+      dataPayload.is_big_message = 'true';
+      dataPayload.attachment = '';
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The notification payload and data payload are created here. However, there's no error handling around this block of code. If any of these operations fail, the entire function will fail. Consider wrapping this block in a try-catch to handle potential errors.
```diff
+    try {
+      const chat = await channel.sendMessage(baseMessage, {skip_push: true});
+      let notificationPayload = {
+        title: `New Message from ${senderInfo?.user?.username}`,
+        body: `${message.substring(0, 100)}`
+      };
+      let dataPayload = {
+        channel_id: channelId,
+        user_id: req.userId,
+        messages_id: chat.message.id,
+        message,
+        message_schema: currentMessageType,
+        type: 'message.new',
+        status: 'sent',
+        is_big_message: 'false',
+        attachment: JSON.stringify(attachments) ?? '',
+        created_at: chat.message.created_at,
+        is_annoymous: 'false'
+      };
+      const dataSizeInBytes = Buffer.byteLength(JSON.stringify(dataPayload), 'utf8');
+      const dataSizeInKilobytes = dataSizeInBytes / 1024;
+      if (dataSizeInKilobytes > 4) {
+        dataPayload.is_big_message = 'true';
+        dataPayload.attachment = '';
+      }
+    } catch (error) {
+      console.error('Error creating notification payload:', error);
+    }
-    const chat = await channel.sendMessage(baseMessage, {skip_push: true});
-    let notificationPayload = {
-      title: `New Message from ${senderInfo?.user?.username}`,
-      body: `${message.substring(0, 100)}`
-    };
-    let dataPayload = {
-      channel_id: channelId,
-      user_id: req.userId,
-      messages_id: chat.message.id,
-      message,
-      message_schema: currentMessageType,
-      type: 'message.new',
-      status: 'sent',
-      is_big_message: 'false',
-      attachment: JSON.stringify(attachments) ?? '',
-      created_at: chat.message.created_at,
-      is_annoymous: 'false'
-    };
-    const dataSizeInBytes = Buffer.byteLength(JSON.stringify(dataPayload), 'utf8');
-    const dataSizeInKilobytes = dataSizeInBytes / 1024;
-    if (dataSizeInKilobytes > 4) {
-      dataPayload.is_big_message = 'true';
-      dataPayload.attachment = '';
-    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->